core/src/main/kotlin/com/bridge/ouroboros/compose/LoopState.kt

```diff
@@ -81,13 +88,9 @@ class LoopStateViewModel<MODEL : Any, EVENT : ActionableEvent<MODEL, EFFECT>, EF
     initialEffects: Set<EFFECT>,
     effectState: EFFECT_STATE,
     crashHandler: CrashHandler = {
-        Log.e(
-            "OuroborosEffects",
-            "Error occured in Ouroboros Effect",
-            it
-        )
+        println("OuroborosEffect - Error occured in Ouroboros Effect: $it")
```

Comment:
Are we sure about that (changing Log to print)?

Reply:
Log only works on android, it won't work on iOS or web

Reply:
But so do ViewModel and a lot of stuff, so there is a long way ahead